nucypher/blockchain/eth/token.py

```diff
@@ -296,6 +296,14 @@ def time_remaining(self, slang: bool = False) -> Union[int, str]:
             result = delta
         return result
 
+    def describe(self) -> Dict[str, str]:
+        start_datetime = self.start_datetime.local_datetime().strftime("%b %d %H:%M %Z")
+        data = dict(index=self.index,
+                    value=str(self.value),
+                    remaining=self.periods_remaining,
+                    enactment=start_datetime)
+        return data
```

Comment:
I think will be useful to add here `final_period` datetime

Reply:
:heavy_check_mark: